packaging/linux/Dockerfile

```diff
@@ -9,8 +9,10 @@ MAINTAINER Keybase <admin@keybase.io>
 #   - 'curl' and 'wget' are for downloading Go and Node
 #   - 'build-essential' pulls in gcc etc., which Go requires.
 #   - python and pip for recent versions of s3cmd
+#   - libc6-dev-i386 for the i386 cgo part of the build
```

Comment:
Yes I think that's covered by `build-essential`. The AMD64 part of the build comes before i386, but only the latter has been seeing errors since adding cgo.